BattleshipFrontend/src/app/board/ship/ship.component.html

```diff
@@ -0,0 +1,23 @@
+<div
+  id="{{ name }}"
+  class="ship-box"
+  [ngStyle]="{
+    'width': isVertical ? (cellPixels + 'px') : (cellPixels * size + 'px'),
+    'height': isVertical ? (cellPixels * size + 'px') : (cellPixels + 'px')
+  }">
+  <div class="ship-draggable-wrapper">
+    <div
+      *ngFor="let i of [0, 1, 2, 3, 4].slice(0, size)"
+      class="ship-box-cell"
+      [ngClass]="{
+        'ship-box-cell-large': boardSize === 10 || boardSize === 15,
+        'ship-box-cell-medium': boardSize === 20 || boardSize === 25,
+        'ship-box-cell-small': boardSize === 30
```

Comment:
why 10 or 15, 20 or 25, 30?
instead of all separated? 
just an info

Reply:
Because the pixels specified for both width and height in the ship-box-cell-large class were ok when the board had 10 or 15 cells, but they started to look a little too big when there where 20 or 25 cells.... And for 30 cells there's no right size hahaha a 30x30 board is just too big! But i made it 5px smaller than the ship-box-cell-medium and i think it looks ok.